### ISSUE TITLE
Request for new video device not proceeding

### ISSUE THREAD

**Issue by roden-roldan-work1** [aws/amazon-chime-sdk-js#762]
**Timeline**
* 2020-10-02T07:03:07.757Z - Started the audiovideo
* 2020-10-02T07:03:18.956Z - User turns on microphone and camera

**Describe the bug**
There seems to be an issue when a particular user turns on the camera and unmutes the microphone. We've already confirmed that there should be no permission issues (in fact, the test on https://app.chime.aws/check shows `ready` for all items), and that the cam/mic shouldn't be in use at that time. 

However, the chime sdk returns the ff for audio (see logs):
```
NotReadableError: Could not start audio source
PermissionDeniedByBrowser
```

and oddly enough, it returns nothing on video. When we call `chooseVideoInputDevice`, the SDK was able to log `requesting new video device with constraint`, but it seems to either die or hang at that point. From what I'm seeing, the SDK should either be logging `failed to get video device for constraint` when there's an error (just like what it did to audio), or a `got video device for constraint` when it succeeds.

What could we do to proceed with unmuting the audio / turning on the camera?

See the logs from start until after the point of error: [SDK logs.txt](https://github.com/aws/amazon-chime-sdk-js/files/5337789/SDK.logs.txt)

**Platform**
 - OS: Windows 10
 - Browser: Chromium
 - Browser Version: 79.0.3945
 - Amazon Chime SDK GitHub Commit: Using chime-sdk-js 1.15.0 (ccfd3c95a1ec3b9a4ddc91595360597bbba3619e)

**Comment by Oladian** [comment]
Same here. When I turn off the camera to turn in again it works, but when I want to change between the cameras on iOS nothing happens. Works in Android and PC, but seems Safari needs to reset or stop the data stream or something like that. I'm not sure why is happening this.

**Comment by p-foucht** [comment]
This *may* be fixed in `v1.20.0` that was just released yesterday. Could you update and let me know if the issue persists?

If it does persist, it doesn't look like a permission issue from looking at the logs (as you've said). Doing a google search shows a fair amount of results around out of date drivers and issues with the "Conexant" microphone, so I would recommend trying these webrtc apps - [one](https://webrtc.github.io/samples/src/content/devices/input-output/), [two](https://appr.tc/) - to see if it is reproducible there. If the issue is still present, it's likely a problem specific to their hardware.

**Comment by devalevenkatesh** [comment]
Hello ,

We are closing this issue. Please feel free to re-open or file a new issue if you need more help.
